src/main/java/org/broadinstitute/consent/http/service/DacService.java

```diff
@@ -141,11 +141,21 @@ public Integer createDac(String name, String description) {
         return dacDAO.createDac(name, description, createDate);
     }
 
+    public Integer createDac(String name, String description, String email) {
+        Date createDate = new Date();
+        return dacDAO.createDac(name, description, email, createDate);
+    }
+
     public void updateDac(String name, String description, Integer dacId) {
         Date updateDate = new Date();
         dacDAO.updateDac(name, description, updateDate, dacId);
     }
 
+    public void updateDac(String name, String description, String email, Integer dacId) {
+        Date updateDate = new Date();
+        dacDAO.updateDac(name, description, email, updateDate, dacId);
+    }
+
```

Comment:
do we need separate code paths for `with email` and `without email`? it seems like going forward we should always use the `with email` code path, so maybe getting ride of the non-email ones would make sense.

Reply:
Yes, I do think we need both right now. If we determine that we want to make email required, we can consolidate them.

Reply:
Ah ok, I was under the impression it was required.